src/providers/getProvider.ts

```diff
@@ -1,14 +1,24 @@
 import { Logger } from '@map-colonies/js-logger';
 import { Tracer } from '@opentelemetry/api';
-import { ProviderConfig, ProviderManager, ProvidersConfig } from '../common/interfaces';
+import { S3Client } from '@aws-sdk/client-s3';
+import { NFSConfig, ProviderConfig, ProviderManager, ProvidersConfig, S3Config } from '../common/interfaces';
 import { NFSProvider } from './nfsProvider';
 import { S3Provider } from './s3Provider';
 
+const nfsProviderName = 'nfs';
```

Comment:
I think it is overkill

Reply:
so what? i don't think i should revert it - its already done...